api/app/services/ingestions/strategy/manifest/text_section.rb

```diff
@@ -19,7 +19,8 @@ def attributes
             source_identifier: source_identifier,
             name: name,
             kind: kind,
-            position: @position
+            position: @position,
+            full_source_identifier: Digest::MD5.hexdigest(@source[:source_path])
```

Comment:
This is my solution for preserving the full path and matching it to the manifest's `start_section_identifier`.  Open to ideas for other approaches.